### DISCUSSION TITLE
Is the Galaxy Brain Badge Still Obtainable in this discussion?

### DISCUSSION THREAD

**Discussion by neeemoooo** [Schweinepriester/achievement-testarea#58] [Q&A]
Hey everyone,
I was wondering if the Galaxy Brain badge is still available for earning in GitHub Discussions. I haven't seen much information about it lately, and I wanted to check if it’s still a feature that can be unlocked. Any insights or updates on how to achieve it would be appreciated! Thanks in advance!

**Comment by giangeralcus** [comment]
Yes, it's still obtainable. What actually counts, based on how it works today:

1. You answer a question in a **Q&A** discussion.
2. The **thread author** (or someone with the accept role) marks your reply as the answer.

Two gotchas people run into:
- **Self-accepting your own answer doesn't count.** You can create a discussion, reply, and mark it accepted, and it will show an accepted answer, but the badge computation excludes those. The acceptance has to come from someone else.
- Replying in `General` or `Show and tell` categories doesn't count — it must be in a **Q&A** category discussion.

So the practical path: answer genuinely useful, unanswered questions here (like the Laravel ones) and hope the author accepts. It's slow, but that's by design.
